enhancements/machine-config/certificate-authorities-for-image-registries.md

```diff
@@ -0,0 +1,186 @@
+---
+title: certificate-authorities-for-image-registries
+authors:
+  - dmage
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - @sinnykumari
+approvers:
+  - @sdodson 
+  - @mrunalp 
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: 2022-12-01
+last-updated: 2022-12-01
+tracking-link:
+  - https://issues.redhat.com/browse/MCO-499
```

Comment:
MCO-499 track mainly work from MCO side. I think we should also have here jira card linked here that covers image registries side of work.